### ISSUE TITLE
Show contributions for private repositories

### ISSUE THREAD

**Issue by dav-s** [IonicaBizau/ghcal#4]
On the GitHub calender, contributions are included from private repositories you have access to. This would mean that if a friend and I contribute to a certain private repo, his contributions would be visible on his calender to me and I would also be able to view my own contributions on my calender.

 This would require the user to log in or provide an API key.

**Comment by IonicaBizau** [comment]
True. However, the there is no official API for this calendar. Maybe we can hack it by passing some authorization headers.

**Comment by IonicaBizau** [comment]
@izuzak Is there a way to get the private contributions in this calendar? I tried:

``` js
curl --user "$USER:<token>" -s https://github.com/users/$USER/contributions
```

(`echo $USER` outputs `ionicabizau`)

However, I see no difference between this authenticated call and the others. Thanks! :smile:

**Comment by IonicaBizau** [comment]
This seems to work actually, but it requires to copy the cookie from the browser:

``` sh
curl --cookie 'Cookie: logged_in=yes; user_session=<user_session_id>;' -s https://github.com/users/$USER/contributions
```

**Comment by izuzak** [comment]
> @izuzak Is there a way to get the private contributions in this calendar?

Using a token to authenticate with github.com doesn't work -- github.com uses sessions.

> This seems to work actually, but it requires to copy the cookie from the browser:

Exactly. They're your _private_ contributions, so you can see them if you authenticate as yourself (with your own session cookie).

**Comment by IonicaBizau** [comment]
Since I :ship: `2.0.0`, we can pass the cookie value:

``` sh
$ ghcal -u izuzak -c 'the cookie header'
```

Implemented in #9. :dizzy:

**Comment by IonicaBizau** [comment]
And using it as library is also possible. It will return an array of arrays in the callback function. :sparkle: 

``` js
var GhCal = require("ghcal");

GhCal({
    username: "izuzak"
   , cookie: "the cookie value"
}, function (err, data) {
    // Do something with the JSON response
    // [..., ["2015-04-26", 22], ...]
    console.log(err || data);
});
```
